spec/spec_helper.rb

```diff
@@ -58,6 +58,13 @@
   Capybara.register_driver :selenium do |app|
     Capybara::Selenium::Driver.new(app, browser: :chrome)
   end
+elsif ENV['WEBDRIVER'] == 'firefox'
+  require 'selenium-webdriver'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.